script/Arcueid/Day 2/02_00_ARC02_3_1.txt

```diff
@@ -11,6 +11,7 @@ I promised Akiha that I'd be back before sunset, after all.
 [sha:d0fa7ff4a138ef1278b9533c12bda648ee9a08e2]{
 -- Page 175, Offset 3055.
 -- 連続殺人事件なんてニュースを鵜呑みにする訳ではないが、人生、何があるか分からない。
+// serial killings or serial murders?
```

Comment:
They're neck and neck on ngrams by the look of it. I guess we use killings more.

Reply:
are we choosing one term or we'll stick to using both?

Reply:
a little variety probably fine